docs/platform/what-is-tizen/overview.md

```diff
@@ -48,6 +48,37 @@ The most important strengths that enable Tizen to move to the next level are IoT
   ![IoTivity for Connectivity](media/about_tizen_6.png)
 
 
+## AI and Machine Learning
+### On-Device & Among-Device AI Pipelines for Inference
+
+NNStreamer (https://github.com/nnstreamer/nnstreamer) is a set of GStreamer plugins that allow GStreamer developers to adopt neural network models easily and efficiently and neural network developers to manage neural network pipelines and their filters easily and efficiently.
+
+- Provide neural network framework connectivities (e.g., tensorflow, caffe) for GStreamer streams.
+  - Efficient Streaming for AI Projects: Apply efficient and flexible stream pipeline to neural networks.
+  - Intelligent Media Filters: Use a neural network model as a media filter / converter.
+  - Composite Models: Multiple neural network models in a single stream pipeline instance.
+  - Multi Modal Intelligence: Multiple sources and stream paths for neural network models.
+- Provide easy methods to construct media streams with neural network models using the de-facto-standard media stream framework, GStreamer.
+  - GStreamer users: use neural network models as if they are yet another media filters.
+  - Neural network developers: manage media streams easily and efficiently.
+
+### On-Device Training & AI Service Personalization
+
+NNTrainer is a software framework for training neural network models on embedded devices which have relatively limited resources. Rather than training whole layers of a network from scratch, NNTrainer finetunes the neural network model on device with user data for the personalization. Using NNtrainer. It is based on the independent open source project NNTrainer(https://github.com/nnstreamer/nntrainer) and provides various applications.
```

Comment:
Fixed article usage in the sentence. 
The line _**“Using NNtrainer.”**_ seems incomplete to me here. Should this be a new bullet point? Please look into this. 
```suggestion
NNTrainer is a software framework for training neural network models on embedded devices which have relatively limited resources. Rather than training whole layers of a network from scratch, NNTrainer finetunes the neural network model on a device with user data for personalization. Using NNtrainer. It is based on the independent open source project NNTrainer (https://github.com/nnstreamer/nntrainer) and provides various applications.
```